services/nvidia-gpu-operator/metadata.yaml

```diff
@@ -11,23 +11,9 @@ certifications:
-  Effectively identify failures, performance degradations, power inefficiencies and their root causes.
-
-  ### GPU Telemetry
-  Gather rich set of GPU telemetry to explain job behavior, identifying opportunities to drive utilization and efficiencies, and determining root causes of potential application performance issues.
-
-  ### Active GPU Health Monitoring
-  Use low-overhead, non-invasive health monitoring while jobs are running without impact to application behavior and performance.
-
-  ### Integration with Management Ecosystem
-  Easily deploy a DCGM based monitoring solution in a Kubernetes cluster environment. Out of the box integration with various ISV solutions such as Bright Cluster Manager, IBM Spectrum LSF and open-source tools such as Prometheus, collected.
+  The NVIDIA GPU Operator uses the operator framework within Kubernetes to automate the management of all NVIDIA software components needed to provision GPU. These components include the NVIDIA drivers (to enable CUDA), Kubernetes device plugin for GPUs, the NVIDIA Container Toolkit, automatic node labelling using GFD, DCGM based monitoring and others.
 
   ## More Information
-  - [developer.nvidia.com/dcgm](https://developer.nvidia.com/dcgm)
-  - [NVIDIA DCGM Documentation](https://docs.nvidia.com/datacenter/dcgm/)
-  - [NVIDIA DCGM - GitHub](https://github.com/NVIDIA/DCGM)
+  - [NVIDIA GPU Operator Documentation](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/overview.html)
+  - [NVIDIA GPU Operator - GitHub](https://github.com/NVIDIA/gpu-operator)
 icon: PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMDAiIGhlaWdodD0iMzAwIj48cGF0aCBkPSJNMTE5LjA4IDExNy42NzF2LTE0LjQzM2w0LjI1NS0uMjJjMzkuNDY5LTEuMjM5IDY1LjM1OCAzMy45MTUgNjUuMzU4IDMzLjkxNXMtMjcuOTY0IDM4Ljg0Mi01Ny45NDggMzguODQyYy00LjMxNiAwLTguMTg2LS42OTktMTEuNjY1LTEuODY2VjEzMC4xNWMxNS4zNjQgMS44NTUgMTguNDUyIDguNjQyIDI3LjY4OSAyNC4wMzlsMjAuNTQ0LTE3LjMyM3MtMTQuOTk1LTE5LjY2OC00MC4yNzMtMTkuNjY4Yy0yLjc1MiAwLTUuMzg0LjE5My03Ljk2LjQ3M20wLTQ3LjY3MXYyMS41NTdhOTAuMTY1IDkwLjE2NSAwIDAgMSA0LjI1NS0uMjUzYzU0Ljg4OC0xLjg1IDkwLjY0MSA0NS4wMDcgOTAuNjQxIDQ1LjAwN3MtNDEuMDcxIDQ5Ljk0NS04My44NTkgNDkuOTQ1Yy0zLjkxOSAwLTcuNTkxLS4zNjMtMTEuMDM3LS45NzR2MTMuMzI3YzIuOTQ1LjM3NCA2LjAwNi41OTQgOS4xOTMuNTk0IDM5LjgxNiAwIDY4LjYxMS0yMC4zMzUgOTYuNDk4LTQ0LjQwMSA0LjYxOCAzLjcwNSAyMy41NDQgMTIuNzA1IDI3LjQ0MSAxNi42NTItMjYuNTE2IDIyLjE5NS04OC4zMDIgNDAuMDg2LTEyMy4zMjggNDAuMDg2LTMuMzc0IDAtNi42MjItLjIwNC05LjgwNC0uNTEydjE4LjcyN2gxNTEuMzM3VjcwSDExOS4wOHptMCAxMDMuOTA4djExLjM3M2MtMzYuODMyLTYuNTYyLTQ3LjA1NS00NC44NDctNDcuMDU1LTQ0Ljg0N3MxNy42ODctMTkuNTkxIDQ3LjA1NS0yMi43NjJ2MTIuNDc5Yy0uMDI3IDAtLjAzOC0uMDA2LS4wNjEtLjAwNi0xNS40MDgtMS44NS0yNy40NTIgMTIuNTQ1LTI3LjQ1MiAxMi41NDVzNi43NDkgMjQuMjQzIDI3LjUxMyAzMS4yMTdtLTY1LjQxMy0zNS4xMzFzMjEuODI2LTMyLjIwOCA2NS40MTMtMzUuNTM5di0xMS42OEM3MC44MDMgOTUuNDI3IDI5IDEzNi4zMSAyOSAxMzYuMzFzMjMuNjc2IDY4LjQ1MiA5MC4wOCA3NC43MTZ2LTEyLjQxOWMtNDguNzI4LTYuMTMyLTY1LjQxMy01OS44MzEtNjUuNDEzLTU5LjgzMSIgZmlsbD0iIzc3YjkwMCIvPjwvc3ZnPg==
```

Comment:
i think you need a new icon

Reply:
hmm maybe it's ok to use the same, looks like the gpu operator https://catalog.ngc.nvidia.com/orgs/nvidia/containers/gpu-operator uses the same

<img width="210" alt="image" src="https://user-images.githubusercontent.com/5897740/196831632-4a6a8f29-59ee-4fb2-b237-50f593c52b1c.png">
this is what it is now (copied from old nvidia app)

Reply:
<img width="142" alt="image" src="https://user-images.githubusercontent.com/5897740/196831704-6cd1c080-e7c7-46b4-ad28-6e47f0752383.png">

Unless we wanted the name. I'll ask UX

Reply:
synced with UX, leaving it as is